l10n_ch_invoice_reports/models/mail_template.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import base64
+
+from odoo import _, models
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    def generate_email(self, res_ids, fields=None):
+        """ Method overridden in order remove QR/ISR payslips and use
+            Invoice report with payslip generated by this module
+        """
+        rslt = super(MailTemplate, self).generate_email(res_ids, fields)
+
+        multi_mode = True
+        if isinstance(res_ids, int):
+            res_ids = [res_ids]
+            multi_mode = False
+
+        res_ids_to_templates = self.get_email_template(res_ids)
+        for res_id in res_ids:
+            related_model = self.env[self.model_id.model].browse(res_id)
+
+            if related_model._name == "account.move":
+                related_model.attachment_ids = False
```

Comment:
we cannot zero all attachments belonging to the model here, that can be not reported as just attachments loaded by the user the reports created in the super method should be in rslt[res_id]["attachments"], they can be removed